test/unit/test_sap_cli_apirelease.py

```
@@ -0,0 +1,544 @@
+class TestApiStateSet(unittest.TestCase):
+
+    def test_set_success_with_info(self):
+        conn = Connection([
+            _resp(API_RELEASE_RESPONSE_XML),
+            _val_resp(VALIDATION_RESPONSE_INFO_XML),
+            _resp(SET_API_RELEASE_RESPONSE_XML),
+        ])
+        group = _make_group()
+        console = BufferConsole()
+        args = _parse(group, 'apistate', 'set', 'c1', 'I_STATISTICALKEYFIGURECAT',
+                       '--state', 'Released', '--corrnr', 'C50K000061')
```

Flake8 (7.3.0)

[error] continuation line over-indented for visual indent

(E127)